extension/custom.css

```diff
@@ -11,3 +11,38 @@ a[href^="github-mac"] {
 .reponav a[href$="/pulse"] {
 	display: none !important;
 }
+
+/* alternate monospace font stack */
+code,
+kbd,
+pre,
+tt,
+.ace_editor.ace-github-light,
+.blame-sha,
+.blob-code-inner,
+.blob-num,
+.branch-name,
+.commit .sha,
+.commit .sha-block,
+.commit-desc pre,
+.commit-ref,
+.commit-tease-sha,
+.default-label .sha,
+.export-phrase pre,
+.file-editor-textarea,
+.file-info,
+.gollum-editor .expanded textarea,
+.gollum-editor .gollum-editor-body,
+.hook-delivery-guid,
+.hook-delivery-response-status,
+.input-monospace,
+.news .alert .branch-link,
+.oauth-app-info-container dl.keys dd,
+.quick-pull-choice .new-branch-name-input input,
+.tag-references>li.commit,
+.two-factor-secret,
+.upload-files .filename,
+.url-field,
+.wiki-wrapper .wiki-history .commit-meta code {
+	font-family: "Source Code Pro", "Droid Sans Mono", "Operation Mono", Consolas, "Liberation Mono", Menlo, Courier, monospace !important;
```

Comment:
Also add a comment about the ordering. That it's sorted by least popular so you're most likely to get the font you want.